examples/plotting/file/sprint.py

```diff
@@ -75,21 +79,23 @@ def selected_name(name, medal, year):
 yaxis = LinearAxis(ticker=yticker, major_tick_in=-5, major_tick_out=10)
```

Comment:
I think the explicit `FixedTicker` is no longer necessary, AFAIK can just pass the list of explicit tick locations and it will be interpreted correctly.

Reply:
TBH the `LinearAxis` could also be dispensed with. More common usage would be like:
```
plot.yaxis.ticker = [....]
plot.yaxis.major_tick_in = -5
plot.yaxis.major_tick_out = 10
```
Then we could remove both model imports

Reply:
It seems like maybe this was done to be able to get the y-axis over on the right on the following line? There's no plot.yaxis.[attr] to use for that, is there?

Reply:
That could be accomplished by passing `y_axis_location="right"` to `figure` instead of explicitly called `add_layout`

Reply:
Just as an aside I mention these all because, for an example in `bokeh/plotting` there is a lot of usage that normally only appears in `bokeh/models` So the other alternative would be to keep all the low level usage but move this example to `bokeh/models` instead.